src/main/java/org/sagebionetworks/web/client/PortalGinModule.java

```diff
@@ -1455,7 +1459,8 @@ protected void configure() {
         bind(SynapseContextPropsProvider.class).to(SynapseContextPropsProviderImpl.class);
         bind(AddToDownloadListV2.class).to(AddToDownloadListV2Impl.class);
         bind(OpenDataView.class).to(OpenDataViewImpl.class);
-
         bind(QueryClientProvider.class).to(QueryClientProviderImpl.class).in(Singleton.class);
+        bind(IntendedDataUseReportWidgetView.class).to(IntendedDataUseReportWidgetViewImpl.class);
+        bind(DialogView.class).to(Dialog.class);

```

Comment:
Added a general DialogView that can be used in a Presenter to build up the UI programmatically.  In this case, wrapping the IDUReport widget in a modal.